File: 240715/학점 계산기/credit-calculator.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        double sum = 0;

        for(int i = 0; i <n; i++){
            sum += sc.nextDouble();
        }

        double avg = sum / n;

        System.out.println(Math.round(avg * 10) / 10.0);

        if(avg >= 4.0){
            System.out.println("Perfect");
        }
        else if(avg >= 3.0){
            System.out.println("Good");
        }
        else{
            System.out.println("Poor");
        }

    }
}

File: 240715/학점 계산기/credit-calculator.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split('\n');

const n = Number(input[0]);
const scores = input[1].split(' ').map(Number);

const sum = scores.reduce((acc, cur) => acc + cur, 0);

console.log((sum / n).toFixed(1));

const calculateScore = (score) => {
    if(score >= 4.0){
        return 'Perfect';
    }else if(score >= 3.0){
        return 'Good';
    }else{
        return 'Poor';
    }
}

console.log(calculateScore(sum / n));